public/index.html

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
```

Comment:
should we add `lang="en"` on the `<html>` tag as a general good practice to adhere to if the default language is english?